test/collaborative_filtering/__init__.py

```diff
@@ -0,0 +1,7 @@
+import sys
+import os
+import site
+
+cwd = os.path.abspath(sys.path[0])
+collaborative_filtering_dir = os.path.join(cwd, "..", "collaborative_filtering")
+site.addsitedir(collaborative_filtering_dir)
```

Comment:
Would this resolve all importing errors from other packages? If so, should we add this to all __ini__ files?

Reply:
You need to modify, how many directories you need to go up. But you would just end up doing the same thing over and over. So using this trick only where it is needed, is sufficient in my opinion.